src/libs/actions/Policy.ts

```diff
@@ -2487,6 +2531,78 @@ function clearCategoryErrors(policyID: string, categoryName: string) {
     });
 }
 
+/**
+ * Accept user join request to a workspace
+ */
+function acceptJoinRequest(reportId: string, accountID: string, adminsRoomMessageReportActionID: string) {
+    const choice = 'accept';
+    const optimisticData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportId}`,
+            value: {
+                [adminsRoomMessageReportActionID]: {
+                    originalMessage: {choice},
+                },
+            },
+        },
+    ];
+
+    const failureData: OnyxUpdate[] = [
+        {
+            onyxMethod: Onyx.METHOD.MERGE,
+            key: `${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportId}`,
+            value: {
+                [adminsRoomMessageReportActionID]: {
+                    originalMessage: {choice: ''},
+                },
+            },
+        },
+    ];
+
+    const parameters = {
+        requests: [{policyID: {accountID, adminsRoomMessageReportActionID}}],
```

Comment:
The correct shape here is as follows. We have to pass the policyID value, not `policyID` as string

```
JSON.stringify({[policyID]: {requests: [{accountID, adminsRoomMessageReportActionID}]}}),
```